src/store/getters.ts

```diff
@@ -531,7 +531,7 @@ export const getEntityTypesBC = (state: StateIF): EntityI[] => {
     return EntityTypesBcData
   } catch (err) {
     console.error('entityTypesBC() =', err) // eslint-disable-line no-console
-    return EntityTypesBcData
+    return []
```

Comment:
I carried over this change from my previous (closed) PR.

The old code masked any errors by returning what looks like legitimate data. I think it's better to return "nothing" as that will stand out during testing and we can fix any problems.